qcodes/dataset/data_set.py

```diff
@@ -1141,14 +1141,10 @@ def get_data_as_xarray(self,
             warnings.warn('Independent parameter setpoints are not equal. Check concatenated output carefully.')
 
         xds = xr.Dataset(data_arrs)
-        for dim in xds.dims:
```

Comment:
Why remove this dimension info?

Reply:
I might be mistaken but I think this is handled by `xds = xr.Dataset(data_arrs)`. At least when I'm viewing a create xarray dataset object after this change the dimension info still shows up and looks right. ''